### PR TITLE
Add bot

### LINKED ISSUE #231: Clusterpedia robot for clusterpedia repository
### What would you like to be added?

Add clusterpedia robot to help to review code and pull request 

### Why is this needed?

Maintainer will merge the pull request after the pipeline succeed manually although maintainers and reviewers review and approve some small pull request at very early time. 
I think we can introduce robot for improving efficiency in `code review` and `merge request`

### PR DESCRIPTION
**Pull request by wzshiming**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #231

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by wzshiming** [comment]
/cc @Iceber
PTAL

File: .github/workflows/bot.yml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    lifecycle
+    retitle
+
+  # This plugins is for in the REVIEWERS environment variable
+  REVIEWERS_PLUGINS: |-
+
+  # This plugins is for in the APPROVERS environment variable
+  APPROVERS_PLUGINS: |-
+    label
+    merge
```
Comment (Iceber) [inline .github/workflows/bot.yml:45]: We may not need to use command to merge for now

Comment (wzshiming) [reply .github/workflows/bot.yml]: Done

File: .github/workflows/bot.yml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+          MESSAGE: ${{ github.event.pull_request.body }}
+          ISSUE_NUMBER: ${{ github.event.pull_request.number }}
+          AUTHOR_ASSOCIATION: ${{ github.event.pull_request.author_association }}
+          ISSUE_KIND: pr
+          TYPE: created
+          GREETING: |-
+            Hi @${{ github.event.pull_request.user.login }},
+            Thanks for your pull request!
+            If the PR is ready, use the `/auto-cc` command to assign Reviewer to Review. 
+            We will review it shortly.
```
Comment (Iceber) [inline .github/workflows/bot.yml:107]: The greeting message feels like it should only be sent to non-members，What do you thing?

Comment (Iceber) [reply .github/workflows/bot.yml:107]: The greeting message of issue is also

**Review by Iceber — APPROVED** [review]
LGTM, Thanks to @wzshiming

**Pull request merged**